src/main/java/noppes/npcs/scripted/ScriptWorld.java

```diff
@@ -93,6 +102,22 @@ public String getSignText(int x, int y, int z){
 		return null;
 	}
 	
+	public void setBlock(int x, int y, int z, Block block){
+		if(block == null || block.isAir(world, x, y, z)){
+			removeBlock(x, y, z);
+			return;
+		}
+		world.setBlock(x, y, z, block);
+	}
```

Comment:
I don't think we need a setBlock function that takes an MC Block class, we don't use those as parameters for functions unless the objective is to get a scripted object back.

Reply:
Actually, other than this everything looks good! I'll be merging as soon as everyone else is on board, then making the additions/changes needed in these comments myself.

Awesome job @1whohears!!!

Reply:
Removed this function. Thanks for taking the time to review my code @c0508383